generators/Input/ConfigFile.cs

```diff
@@ -2,28 +2,36 @@
 using System.IO;
 using System.Linq;
 using Newtonsoft.Json;
+using Generators.Output;
 
 namespace Generators.Input
 {
     public static class ConfigFile
     {
         private const string ConfigFilePath = "../config.json";
 
-        public static IEnumerable<string> GetExerciseNames()
+        public static IEnumerable<ConfigExercise> GetExercises()
         {
             var jsonContents = File.ReadAllText(ConfigFilePath);
             var config = JsonConvert.DeserializeObject<Config>(jsonContents);
-            return config.Exercises.Select(exercise => exercise.Slug).OrderBy(x => x).ToArray();
+            return config.Exercises.OrderBy(x => x.Name).ToArray();
         }
 
         private class Config
         {
             public ConfigExercise[] Exercises { get; set; }
         }
 
-        private class ConfigExercise
+        public class ConfigExercise
         {
+            public string Name { get { return this.Slug.ToExerciseName(); } }
```

Comment:
You can simplify this a little bit: `public string Name => Slug.ToExerciseName();`